### PR TITLE
Ft/wisdom/findjob

### PR DESCRIPTION
**Pull request by Ajinormotor**
## Description

Please include a summary of the changes and the related issue. 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Ajinormotor** [comment]
i have done the necessary job i will be open to correction

File: public/icons/IC_PaginateLeft.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+const IC_PaginateLeft = (props: { className?: string }) => {
+  return (
```
Comment (Ajinormotor) [inline public/icons/IC_PaginateLeft.tsx:4]: hello

**Comment by Ajinormotor** [comment]
done with the changes on the findjob hero, font,size responsiveness filter section state

**Comment by Blossom1-chike** [comment]
> done with the changes on the findjob hero, font,size responsiveness filter section state and also worked on hiring company header, recommended and category

did you commit?
the last changes were committed 2 days ago, can't see any updated ones

**Pull request merged**